students/slawinski_amadeusz/lesson_02_flow_control/3.01_bishop_moves.py

```diff
@@ -0,0 +1,20 @@
+# check if bishop can move to target position
+
+import math
+
+bishop_x = int(input())
+bishop_y = int(input())
+dest_x = int(input())
+dest_y = int(input())
+
+delta_x = bishop_x - dest_x
+delta_y = bishop_y - dest_y
+# time to compare if we move the same amount in both directions
+# to do this just lose direction vector (sign)
+delta_x = math.fabs(delta_x)
```

Comment:
można by to skrócić do: `delta_x = math.fabs(bishop_x - dest_x)`
W pythonie są dwie funkcję zwracające wartość bezwzględną: `abs` i `math.fabs()`
fabs sprawdza czy może zrobić konwersje danych wejściowych do float, jako że nasze dane wejściowe są int, niejest to koniecznie, dlatego lepszym wyborem jest tutaj `abs`.
Więcej informacji o różnicach:
https://stackoverflow.com/questions/10772302/python-abs-vs-fabs